articles/storage/tables/table-storage-how-to-use-powershell.md

```diff
@@ -28,9 +28,14 @@ This how-to article covers common Azure Table storage operations. You learn how
 
 This how-to article shows you how to create a new Azure Storage account in a new resource group so you can easily remove it when you're done. If you'd rather use an existing Storage account, you can do that instead.
 
-The examples require Azure PowerShell module `AzureRM` version 4.4.0 or later. In a PowerShell window, run `Get-Module -ListAvailable AzureRM` to find the version. If nothing is displayed, or you need to upgrade, see [Install Azure PowerShell module](/powershell/azure/azurerm/install-azurerm-ps).
+The examples require Az PowerShell modules `Az.Storage (1.0.3 or greater)` and `Az.Resources (1.2.0 or greater)`. In a PowerShell window, run `Get-Module -ListAvailable Az*` to find the version. If nothing is displayed, or you need to upgrade, see [Install Azure PowerShell module](/powershell/azure/install-az-ps).
 
-[!INCLUDE [requires-azurerm](../../../includes/requires-azurerm.md)]
+> [!IMPORTANT]
+>
+> Using this Azure feature from PowerShell requires that you have the `Az` module installed. Current version of **AzureRmStorageTable**
```

Comment:
The current version of **AzureRmStorageTable** is not compatible with the older **AzureRM** module.